packages/core/trie/nodes.ts

```diff
@@ -72,7 +72,7 @@ export enum NodeType {
 export class TrieNode {
   constructor(
     /** Exactly 512 bits / 64 bytes */
-    public readonly raw: Uint8Array = new Uint8Array(TRIE_NODE_BYTES),
+    public readonly raw: Uint8Array = safeAllocUint8Array(TRIE_NODE_BYTES),
```

Comment:
FYI: In cases like this, when the allocation size is known and/or bounded, we don't necessarily need to use the helper function.